Formula/l/limine.rb

```diff
@@ -0,0 +1,48 @@
+class Limine < Formula
+  desc "Modern, advanced, portable, multiprotocol bootloader and boot manager"
+  homepage "https://limine-bootloader.org"
+  url "https://github.com/limine-bootloader/limine/releases/download/v9.5.4/limine-9.5.4.tar.gz"
+  sha256 "5f4752b5bb01272143506db91b83d46489e9fb2546dfea776df87040bdbcd228"
+  license "BSD-2-Clause"
+
+  depends_on "lld" => :build
+  depends_on "llvm" => :build
+  depends_on "mtools" => :build
+  depends_on "nasm" => :build
+
+  def install
+    # Save the system Clang, so we can use it to build the host limine tool
+    # via the CC configure variable.
+    system_clang = `which clang`
+
+    # Add Homebrew LLVM to the PATH so that it is used for the rest of the
+    # (non-host) bootloader binaries.
+    ENV.prepend_path "PATH", Formula["llvm"].opt_bin
+
+    system "./configure", *std_configure_args, "--enable-all",
+           "CC=#{system_clang}"
```

Comment:
These shouldn't be needed.

```suggestion
    system "./configure", *std_configure_args, "--enable-all"
```